source/table-filters/apply-table-filters.txt

```diff
@@ -1,3 +1,46 @@
 ===================
 Apply Table Filters
-===================
\ No newline at end of file
+===================
+
+
+You can use table filters to migrate specific subsets of your table data.
+Table filters allow you to apply SQL where and row limit clauses
+when pulling data from your table.
+
+Before You Begin
+----------------
+
+Table filters are applied while you are creating mapping rules. See 
+:ref:`<rm-create-mapping-rules>` for more details.
+
+You can apply one filter per table. Each filter can have a SQL where 
+clause, a row limit clause, or both.
+
+Steps
+-----
+
+
+#. From the :guilabel:`Schema model` pane, click the table that you want to apply a filter to.
+#. On the :guilabel:`Filter on` section in the edit mapping rule pane, click :guilabel:`+ Add`.
+#. Under the :guilabel:`SQL query` header, enter a SQL where criteria. The 
+   SQL you specify must be ANSI SQL compliant with your relational database.
+   The following are examples of SQL where clauses used to apply 
+   :guilabel:`SQL query` table filters:
+
```

Comment:
(question)
Do we intend this to be a single, copyable code block? 
Should it instead be a bulleted list of code phrases?

Reply:
I bulleted this out, it looks slightly better.